modules/uhttp/filters.go

```diff
@@ -55,8 +55,7 @@ type contextFilter struct {
 }
 
 func (f contextFilter) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
-	ctx := ulog.NewLogContext(r.Context())
-	next.ServeHTTP(w, r.WithContext(ctx))
+	next.ServeHTTP(w, r)
```

Comment:
Makes sense. We'll task up to remove this, since it's pretty clearly useless now.